src/solc_0.7/AssetGiveaway/AssetGiveaway.sol

```diff
@@ -6,8 +6,8 @@ import "@openzeppelin/contracts/token/ERC1155/IERC1155Receiver.sol";
 import "./ClaimERC1155.sol";
```

Comment:
general comment : let's organise the Claims contract in their own folder 
src/solc_0.7/claims/AssetGiveAway/...
src/solc_0.7/claims/LandGiveAway/...
...